cookiecutter/{{cookiecutter.organization}}-richie-site-factory/template/{{cookiecutter.site}}/src/frontend/scss/extras/settings/_variables.scss

```
@@ -0,0 +1,17 @@
+@import 'bootstrap/scss/functions';
+@import 'bootstrap/scss/variables';
+
+@import 'richie-education/scss/colors/palette';
+// palette overrides
+@import 'colors/palette';
+
+@import 'richie-education/scss/colors/gradients';
+
+@import 'richie-education/scss/colors/schemes';
+
+@import 'richie-education/scss/colors/theme';
+// theme overrides
+@import 'colors/theme';
+
+@import 'colors/richie_upstream_variables';
```

Related to this [comment](https://github.com/openfun/richie/pull/1687#discussion_r853169942)

If we prefix our variables by `!default` within `richie-education/scss/settings/_variables`, I think we could update code as follow :

```suggestion
@import 'richie-education/scss/settings/_variables';
```

---

Ok, but for now we can't use that because the cookiecutter would use the latest Richie release and not what it is in the master branch.

---

Yes, good point !